bundles/org.openhab.binding.wled/src/main/java/org/openhab/binding/wled/internal/WLedHelper.java

```diff
@@ -37,10 +37,7 @@ public static HSBType parseToHSBType(String message, int brightness) {
             int r = new BigDecimal(colors.get(0)).intValue();
             int g = new BigDecimal(colors.get(1)).intValue();
             int b = new BigDecimal(colors.get(2)).intValue();
-            HSBType tmp = HSBType.fromRGB(r, g, b);
-            PercentType brightnessPercent = new PercentType(
-                    new BigDecimal(brightness).divide(BIG_DECIMAL_2_55, RoundingMode.HALF_UP));
-            return new HSBType(tmp.getHue(), tmp.getSaturation(), brightnessPercent);
+            return HSBType.fromRGB(r, g, b);
```

Comment:
This seems to be a straight reversal of #16907?
In which case you should revert the method siugnature from `parseToHSBType(String message, int brightness)` back to `parseToHSBType(String message)` since the brightness argument is once more not required.

Reply:
Good catch, thanks